docs/superpowers/plans/2026-05-28-mcp-timeout-improvements.md

```
@@ -1409,17 +1409,61 @@ git commit -m "docs: add Phase 2 timeout configuration documentation"
 | D-2 | LLM Evaluator 並列化 | ✅ | Task 2 |
 | D-3 | 承認 timeout + bypass | ✅ | Task 3 |
 | E-1 | Chunk 並列化 | ✅ | Task 4 |
-| E-2 | 埋め込みリトライ調整 | ✅ | Task 5 |
+|| E-2 | 埋め込みリトライ調整 | 🔄 | Task 5（未着手） |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)